crates/aggchain-proof-contracts/src/config.rs

```diff
@@ -41,6 +42,12 @@ pub struct AggchainProofContractsConfig {
     /// Caller address for the static calls
     #[serde(default = "default_static_call_caller_address")]
     pub static_call_caller_address: Address,
+
+    // EVM sketch genesis configuration
+    // Default is "mainnet", could be "sepolia", "opmainnet"
+    // or path to a custom genesis file.
+    #[serde(default = "default_evm_sketch_genesis")]
+    pub evm_sketch_genesis: String,
```

Comment:
Nit: This could be an enum to avoid the custom parsing, roughly like this:
```rust
#[derive(Serialize, Deserialize)]
enum EvmSketchGenesis {
    Mainnet,
    Sepolia,
    Opmainnet,
    Linea,
    #[serde(untagged)]
    File(PathBuf),
}
```

But it does not check non-emptiness and file existence, that would still have to be done separately.